internal/responses/responses.go

```diff
@@ -0,0 +1,81 @@
+package responses
```

Comment:
I checked the Handlers, I don't believe this is used for anything but the docs, not actually used in the implementation

Reply:
Yes, hence my remark. Now it looks better.